cmd/juju/machine/add.go

```diff
@@ -266,7 +274,31 @@ func (c *addCommand) getMachineManagerAPI() (MachineManagerAPI, error) {
 }
 
 func (c *addCommand) Run(ctx *cmd.Context) error {
-	var err error
+	var (
+		base series.Base
+		err  error
+	)
+	// Note: we validated that both series and base cannot be specified in
+	// Init(), so it's safe to assume that only one of them is set here.
+	if c.Series != "" {
+		if c.Series == "kubernetes" {
+			ctx.Warningf("using kubernetes as a series flag is deprecated, use --base instead")
```

Comment:
do we need a specific log for `kubernetes` series? can't we just use the log `ctx.Warningf("series flag is deprecated, use --base instead")` outside the if check?

Reply:
I wanted to call out kubernetes specifically here. It's really, really deprecated and should be killed with fire.

Reply:
sure! and thanks for the test!